PackageInfo.g

```diff
@@ -3,7 +3,17 @@ SetPackageInfo( rec(
 PackageName := "kbmag",
 Subtitle := "Knuth-Bendix on Monoids and Automatic Groups",
 Version := "1.5.4dev",
-Date := "01/02/2017",
+Date := "21/12/2017",
+
+##  duplicate these values for inclusion in the manual: 
+##  <#GAPDoc Label="PKGVERSIONDATA">
+##  <!ENTITY KBMAGVERSION "1.54dev">
+##  <!ENTITY KBMAGRELEASEDATE "21/12/2017">
+##  <!ENTITY KBMAGLONGRELEASEDATE "21st December 2017">
+##  <!ENTITY KBMAGTARFILENAME "kbmag-1.54dev.tar.gz">
+##  <!ENTITY KBMAGHTMLFILENAME "kbmag.html">
+##  <!ENTITY KBMAGCOPYRIGHTYEARS "1997-2017">
```

Comment:
`KBMAGCOPYRIGHTYEARS` is used in exactly one place. I see no advantage in turning it into a variable.

I also have doubts about increasing that range to end with 2017 -- none of the authors made recent changes, and the copyright notice only covers actual changes -- you cannot extend copyright claims merely by adjusting a copyright statements. 

In fact, it is not necessary at all to specify a range. From a legal point of view (at least in countries which signed the Berne convention), it suffices to give the date when the work was created, i.e. 1997 in this case; or possibly a list of years in which major changes were made (e.g. 1997,